src/components/Carousel.js

```diff
@@ -650,7 +652,9 @@ class Carousel extends Component {
                             </ul>
                         }
                     </div>
-                    <button type="button" className={klass.ARROW_NEXT(!hasNext)} onClick={this.increment} />
+                    <button type="button" className={klass.ARROW_NEXT(!hasNext)} onClick={this.increment} >
+                        <span className="hidden">Next slide</span>
```

Comment:
You don't need an extra span for this change. You could just add the text as direct child of the button and use css to hide it. Have you considered languages other than English? Making it a prop (with English strings as default value) would be enough to allow consumers to have it translated.

Reply:
Why hiding the text wouldn't be enough? I'm pretty sure that if you apply the same trick you did on the .hidden class to the button, it will work.

Reply:
Yeah you are right. I'm waiting for my PM to give me some time to work on this. Probably for the end of this sprint. Sorry about the delay.